django_app/redbox_app/redbox_core/models.py

```diff
@@ -59,6 +59,10 @@ class File(UUIDPrimaryKeyBase, TimeStampedModel):
     original_file = models.FileField(storage=settings.STORAGES["default"]["BACKEND"])
     user = models.ForeignKey(User, on_delete=models.CASCADE)
     original_file_name = models.TextField(max_length=2048, blank=True, null=True)
+    core_file_uuid = models.UUIDField(null=False)
+
+    class Meta:
+        unique_together = ["user", "core_file_uuid"]
```

Comment:
```suggestion
        unique_together = ["id", "core_file_uuid"]
```
isnt it these two that are "the same"?

Reply:
Nope - the `id` is a UUID, the `user` is a foreign key.

Reply:
yes but isnt it the files that are supposed to match rather than a user and a file?

Reply:
I think we want:
* the `core_file_uuid` to be unique (in its own right)
* a 1:1 relationship between the original real file (`id`) and its embedded file (`core_file_uuid`)
?

Reply:
I think you need:
1. a migration to add a nullable field with a default value
2. a data migration to back populate the new field for all the existing records 
3. a migration to make the new field non nullable

it is (2) that is missing, without it (3) will fail no?

Reply:
Take a look now?